src/content/docs/en/pages/main-menu/reference/observe/real-time-events/real-time-events.mdx

```diff
@@ -41,261 +41,271 @@ You can use Real-Time Events to:
+| Host | Host information sent on the request line. Stores: host name from the request line, or host name from the `Host` request header field, or the server name matching a request. Example: `g1sdetynmxe0ao.map.azionedge.net` |
+| HTTP Referer | Address of the page the user made the request from. Example: `https://example.com` |
+| HTTP User Agent | End user’s application, operating system, vendor, and/or version. Value of the `User-Agent` header. Example: `Mozilla/5.0 (Windows NT 10.0; Win64; x64)` |
+| Request Length | Request length in bytes, including request line, headers, and body. This field is the result of a sum. Example: `167` |
+| Request Method | HTTP request method. Example: `GET` or `POST` |
+| Request Time | Request processing time elapsed since the first bytes were read from the client with resolution in milliseconds. This field is the result of a sum. Example: `1.19` |
+| Request Uri | URI of the request made by the end user, without the host and protocol information and with arguments. Example: `/v1?v=bo%20dim` |
+| Remote Address | IP address of the origin that generated the request. Example: `127.0.0.1` |
+| Remote Port | Port of the origin that generated the request. Example: `8080` |
+| Scheme | Request scheme. Example: `HTTP` or `HTTPS` |
+| Server Protocol | Version of the request protocol. Example: `HTTP/1.1`, `HTTP/2.0`, `HTTP/3.0` |
+| Sent HTTP Content Type | `Content-Type` header sent in the origin’s response. Example: `text/html; charset=UTF-8` |
+| SSL Cipher | Cipher string used to establish TLS connection. Example: `TLS_AES_256_GCM_SHA384` |
+| SSL Protocol | Protocol for an established TLS connection. Example: `TLS v1.2` |
+| Stack Trace | Provides the names of the Rules Engine from your edge application or your edge firewall that are run by the request. Example: `{\\\"edge_firewall\\\":[\\\"Global - Set WAF\\\"]}` |
+| Status | HTTP status code of the request. Example: `200` |
+| Upstream Addr | Client’s IP address and port. Can also store multiple servers or server groups. Example: `192.168.1.1:80`. When the response is `127.0.0.1:1666`, the upstream is [Azion Cells Runtime](/en/documentation/devtools/runtime/overview/). |
+| Upstream Bytes Received | Number of bytes received by the origin’s edge if the content isn’t cached. Example: `8304` |
+| Upstream Bytes Sent | Number of bytes sent to the origin. Example: `2733` |
+| Upstream Cache Status | Status of the local edge cache. Can be: `MISS`, `BYPASS`, `EXPIRED`, `STALE`, `UPDATING`, `REVALIDATED`, `HIT`, or `-` |
+| Upstream Response Time | Time it takes for the edge to receive a default response from the origin in milliseconds, including headers and body. Example: `0.876`. **In case of cache, the response is `-`**. |
+| Upstream Status | HTTP status code of the origin. If a server can't be selected, the variable keeps the 502 (Bad Gateway) status code. Example: `200`. **In case of cache, the response is `-`**. |
```

Comment:
```suggestion
| Upstream Status | HTTP status code received from the origin. If a server can't be selected, the variable keeps the `502 Bad Gateway` status. Example: `200`. **In case of cache, the response is `-`** |
```